### PR TITLE
wee typo plus two tweaks

### PR DESCRIPTION
**Pull request by Mulesoft-Mysti**
I saw a random close parens and removed it, then added a link to the ref doc and cleaned up an intro para that was a little long. It's all at the top of the topic.

### CODE REVIEW

**Comment by dukesphere** [comment]
my only comment would be that we link to the Core types twice, once in the note and another time with the other types

**Comment by Mulesoft-Mysti** [comment]
Great catch, thank you. I was thinking those links went to internal topics. Will fix.

File: mule4-user-guide/v/4.1/dataweave-types.adoc [hunk 1]
```diff
@@ -1,22 +1,24 @@
 = DataWeave Types
 :keywords: studio, anypoint, esb, transform, transformer, format, aggregate, rename, split, filter convert, xml, json, csv, pojo, java object, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping
 
-//TODO: NEED TO MERGE THIS INFO WITH THE CORE TYPES. DISCUSS WITH MDA.
-The built-in DataWeave functions support a number of data types for elements, including arrays, Booleans, objects, dates, times, and many others.
+DataWeave functions operate on data of many types, including arrays, Booleans, objects, dates, and times.
+The types that DataWeave provide are bundled into modules that also contain the related functions. 
 
-NOTE: For a complete reference of core types, visit link:/mule4-user-guide/v/4.1/dw-core-types[Core Types].
+The `dw::Core` module types (core types) are available without having to specify the Core module. Other modules may need to be specified for their functions and types to be available.
```
[the remark this answers is not in the document]
Comment (Mulesoft-Mysti) [reply mule4-user-guide/v/4.1/dataweave-types.adoc]: 1. Why capitalize? The module is named Core, but there's nothing named Core Type, it's just a description?
2. See follow up email.
3. replaced specified, thanks!

File: mule4-user-guide/v/4.1/dataweave-types.adoc [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
 
-== Supported Types
+This topic provides an overview of the commonly used types. A complete reference is available in link:/dw-functions[each module's reference pages].
```
[the remark this answers is not in the document]
Comment (Mulesoft-Mysti) [reply mule4-user-guide/v/4.1/dataweave-types.adoc]: removed

File: mule4-user-guide/v/4.1/dataweave-types.adoc [hunk 3]
```diff
@@ -1,22 +1,24 @@
...
 * link:dw-runtime-types[Runtime Types (dw::Runtime)]
 * link:dw-url-types[URL Types (dw::core::URL)]
 * link:dw-diff-types[Diff Types (dw::core::Diff)]
 * link:dw-timer-types[Timer Types (dw::core::Timer)]
 
-This topic also provides information about behaviors common to many DataWeave types:
+This topic provides descriptions of common patterns of use:
 
 * Conditional elements that use `if` expressions for <<conditional_elements_array, arrays>> and <<conditional_elements_object, objects>>.
 * <<string_interpolation>>
```
Comment (Mulesoft-Mysti) [inline mule4-user-guide/v/4.1/dataweave-types.adoc]: I moved this to the See Also section.

**Pull request merged**